### PR TITLE
Added R code and dialog for Compare Columns

### PR DESCRIPTION
**Pull request by dannyparsons**
@africanmathsinitiative/developers this is ready for review. This is a new dialog for Prepare > Check Data (and the new Options by Context menu), useful for comparing the values in two columns before doing a merge to check the values match, so likely the two columns will be from different data frames. An example is comparing the values in the station columns of the three data frames in the Western Kenya data.

@shadrackkibet I have added to our functionality for getting columns from different data frames. It shouldn't affect how this works in existing climatic dialogs but please check.

### CODE REVIEW

**Comment by dannyparsons** [comment]
Dialog design is not great so any suggestions to improve welcomed:

image

**Comment by shadrackkibet** [comment]
Reviewing and testing this.

**Comment by shadrackkibet** [comment]
This is working great. Just a  small issue to sort.

When i click on save checkbox the save input is empty, i also see ok is enabled at this instance.

File: instat/dlgCompareColumns.vb [hunk 1]
```diff
@@ -1,3 +1,153 @@
...
+'
+' This program is distributed in the hope that it will be useful,
+' but WITHOUT ANY WARRANTY; without even the implied warranty of
+' MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+' GNU General Public License for more details.
+'
+' You should have received a copy of the GNU General Public License 
+' along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+Imports instat
```
Comment (shadrackkibet) [inline instat/dlgCompareColumns.vb]: do we need this import?

**Comment by dannyparsons** [comment]
Thanks, I've fixed those now.

**Comment by shadrackkibet** [comment]
This is now working great for me. Not unless someone else has an issue to report, this can be merged.

**Comment by dannyparsons** [comment]
Thanks, could someone add this to the menu. Prepare > Check Data > and then maybe under Duplicates or further down? @rdstern?

**Comment by shadrackkibet** [comment]
Adding this to the form main.

**Pull request merged**